_includes/lesson_footer.html

```diff
@@ -30,17 +30,17 @@
     <div class="col-md-6 help-links" align="right">
 	{% if page.source %}
 	{% if page.source == "Rmd" %}
-	<a href="{{site.github.repository_url}}/edit/{{ default_branch }}/{{page.path|replace: "_episodes", "_episodes_rmd" | replace: ".md", ".Rmd"}}">Edit on GitHub</a>
+	<a href="{{repo_url}}/edit/{{ default_branch }}/{{page.path|replace: "_episodes", "_episodes_rmd" | replace: ".md", ".Rmd"}}">Edit on GitHub</a>
 	{% endif %}
 	{% else %}
-	<a href="{{site.github.repository_url}}/edit/{{ default_branch }}/{{page.path}}">Edit on GitHub</a>
+	<a href="{{repo_url}}/edit/{{ default_branch }}/{{page.path}}">Edit on GitHub</a>
```

Comment:
It might make sense to display this only when the page is rendered "online"

Reply:
This might create a loophole: if something does not work in the rendered version, it might not indicate a problem.

I guess, it would make more sense if this `Edit on GitHub` link points to GitHub no matter where it is rendered.

Reply:
This is out of the scope of the problem that this PR is addressing. I opened a separate issue (#352) to keep track of it.